pom.xml

```diff
@@ -582,6 +582,59 @@
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
Check if any method to make profile common for 2.2 and 2.3 and pass only specific details though override or parameters.